### PR TITLE
v1.2 Delete & Edit sales record by index

### PR DESCRIPTION
**Pull request by HyperionNKJ**
Implemented delete-sales and edit-sales by index features.
Test code has been written as well. PR is ready for review.

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 1]
```diff
@@ -129,6 +132,12 @@ public Command parseCommand(String userInput) throws ParseException {
         case DisplaySalesCommand.COMMAND_WORD:
             return new DisplaySalesCommandParser().parse(arguments);
```
Comment (AZhiKai) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java]: Consider adding command alias.

File: src/main/java/seedu/address/ui/sales/RecordCard.java [hunk 2]
```diff
@@ -44,7 +46,9 @@ public RecordCard(SalesRecord record, int displayedIndex) {
         date.setText(record.getDate().toString());
         itemName.setText(record.getName().toString());
         quantitySold.setText("Quantity Sold: " + String.valueOf(record.getQuantitySold().toString()));
-        price.setText("Item Price: $" + String.valueOf(record.getPrice().toString()));
+
+        NumberFormat currencyFormatter = NumberFormat.getCurrencyInstance();
+        price.setText("Item Price: " + currencyFormatter.format(record.getPrice().getValue()));
```
Comment (AZhiKai) [inline src/main/java/seedu/address/ui/sales/RecordCard.java]: What's the difference between using `getValue()` and `toString()`? Both method seems to return the same variable.

Comment (HyperionNKJ) [reply src/main/java/seedu/address/ui/sales/RecordCard.java]: getValue() returns the price in float, while toString() returns it as a string.
Without getValue() I'll have to do the following:
Float.parseFloat(record.getPrice().toString())
which I find slightly long-winded. 😄

Comment (AZhiKai) [reply src/main/java/seedu/address/ui/sales/RecordCard.java]: I see, okay! 👍

**Pull request merged**